release.json

```diff
@@ -8,6 +8,14 @@
 		"JMXFETCH_VERSION": "0.27.0",
 		"JMXFETCH_HASH": "52aabf81abb4e42adcb28164b263f01b293dda1c686f845d133ebcd128f8b2ee"
 	},
+	"6.11.0": {
+		"INTEGRATIONS_CORE_VERSION": "6.11.0",
+		"PROCESS_AGENT_VERSION": "6.11.0",
+		"OMNIBUS_SOFTWARE_VERSION": "6.11.0",
+		"OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
+		"JMXFETCH_VERSION": "0.27.0",
+		"JMXFETCH_HASH": "52aabf81abb4e42adcb28164b263f01b293dda1c686f845d133ebcd128f8b2ee"
+	},
```

Comment:
nit: you can typically remove the RCs now.

Reply:
Yep! I've hesitated to do it there or in another PR